Python/sort/comb_sort.py

```diff
@@ -0,0 +1,74 @@
+"""
+CombSort Algorithm Implementation in Python 3
+
+Purpose:
+Comb sort is a relatively simple sorting algorithm.
+Comb sort improves on bubble sort algorithm.
+In bubble sort, distance (or gap) between two compared elements is always one.
+Comb sort improvement is that gap can be much more than 1, in order to prevent
+slowing down by small values at the end of a list.
+
+Time Complexity :
+Worst case complexity is O(n^2)
+Best Case complexity is O(n log n)
+
+Space Complexity: O(1)
```

Comment:
- Add Time and Space Complexity at the end of the Code

Reply:
Added time and space complexity after sample input/output.